### ISSUE TITLE
Tensorboard Log

### ISSUE THREAD

**Issue by GuoxingYan** [TonyXuQAQ/RNGDetPlusPlus#16]
Hello, is it convenient to publish a tensorboard log map during the training process?

**Comment by GuoxingYan** [comment]
In my training, loss_instance_seg is an order of magnitude lower than other losses. Why is the scaling ratio of 1/5 in the paper?

**Comment by TonyXuQAQ** [comment]
1. The script for tensorboard could be found here ```./bash/run_tensorboard.bash```.
2. In our experiments, all loss functions have the same order of magnitude. Loss weight 1/5 was used in early experiments. Please go through the training code for details and you could try other weights to balance loss functions.

**Comment by GuoxingYan** [comment]
ok
